apps/ffdweb-site/src/app/(homepage)/page.tsx

```diff
@@ -6,12 +6,26 @@ import { FFDW_URLS } from '@/constants/siteMetadata'
 import { createMetadata } from '@/utils/createMetadata'
 
 import { CTASection } from '@/components/CTASection'
+import { PageHeader } from '@/components/PageHeader'
+
+import placeholderImage from '@/assets/graphics/placeholder.webp'
```

Comment:
Why is this called a `placeholderImage`?

Reply:
We can address this when we build out the graphics object!

Reply:
Yes, sorry it was intended for testing